tests/cross/src/test/scala/tests/pc/InsertInferredTypeSuite.scala

```diff
@@ -454,6 +454,30 @@ class InsertInferredTypeSuite extends BaseCodeActionSuite {
        |""".stripMargin
   )
 
+  checkEdit(
+    "either",
+    """|object O{
+       |  def <<returnEither>>(value: String) = {
```

Comment:
I can't seem to reproduce the other problems from the issue (wrong rename and position) with the proposed fix in place, so it seems it might only be related to the basic Scala type aliases.